client/analytics/components/report-table/index.js

```diff
@@ -165,12 +165,31 @@ const ReportTable = ( props ) => {
 		}
 
 		if ( data && data.length === totalResults ) {
+			/**
+			 * Filter report table for the CSV download.
+			 *
+			 * Enables manipulation of data used to create the report CSV.
+			 *
+			 * @param {Object} reportTableData - data used to create the table.
+			 * @param {string} reportTableData.endpoint - table api endpoint.
+			 * @param {Array} reportTableData.headers - table headers data.
+			 * @param {Array} reportTableData.rows - table rows data.
+			 * @param {Object} reportTableData.totals - total aggregates for request.
+			 * @param {Array} reportTableData.summary - summary numbers data.
+			 * @param {Object} reportTableData.items - response from api requerst.
+			 */
+			const { headers, rows } = applyFilters( TABLE_FILTER, {
+				endpoint,
+				headers: getHeadersContent(),
+				rows: getRowsContent( data ),
+				totals,
+				summary: getSummary ? getSummary( totals, totalResults ) : null,
+				items,
+			} );
```

Comment:
Rather than duplicate this code block, what do you think about moving it to a reusable function?

Reply:
I was pondering this while drifting off to sleep last night, wondering what reason I had for duplicating it. Will do :-D

Reply:
@jeffstieler Done in 320b5e80, thanks